ecocode-rules-specifications/src/main/rules/EC4/python/EC4.asciidoc

```diff
@@ -1,24 +0,0 @@
-When function calls global variables, a lot a CPU cycles is consumed.
```

Comment:
same review feedback like PHP part : I don't agree to delete Python EC4 because not deprecated and still in usage (check "EC4" rule in array above)